### ISSUE TITLE
508-defect-3 [COGNITION]: Introduction page, links should be styled and written as links

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#22846]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

<hr/>

## Point of Contact

**VFS Point of Contact:** Josh Kim

## Details

"Start your application without signing in" is _styled_ as a link but is written as a `button` which is materially dishonest. Users may become frustrated that they are unable to open a new tab. Screen reader users may be confused that the text seems more like a link despite being announced as a button.

For more on this long-tail effort, check out our [buttons vs. links Mural board](https://app.mural.co/t/vsa8243/m/vsa8243/1610052905994/253723ef4b2ee9a726f538876a257df94d468db1) and this [Slack thread discussing _why_ material honesty is so important](https://dsva.slack.com/archives/C8E985R32/p1610659601009200?thread_ts=1610652897.002600&cid=C8E985R32).

"A few years at an accessibility event, a blind man was telling a story about how he was on the phone with someone who was trying to help him sign up for a service. The person on the phone told him to click the “sign up button” and he couldn’t find it because it was a link. He told the story with a lot more gravity than I can, but that was the point where I realized how important the human-human communication part of accessibility is, it’s not just machine to human." - Tim Wright

## Acceptance Criteria

- [ ] "Start your application without signing in" is written with an `a` tag

## Environment

* Operating System: all
* Browser: any
* Screenreading device: any
* Server destination: staging

## Solution (if known)

Use `a` instead of `button` 

## WCAG or Vendor Guidance (optional)

* [Adam Silver: But sometimes buttons look like links](https://adamsilver.io/articles/but-sometimes-buttons-look-like-links/)
* [VA Design System guidance on buttons](https://design.va.gov/components/buttons#guidance)

## Screenshots or Trace Logs

<img width="1323" alt="Screen Shot 2021-04-07 at 12 16 28 PM" src="https://user-images.githubusercontent.com/14154792/113900114-7eeb3180-979b-11eb-9ae7-d22b0a280436.png">

**Comment by micahchiang** [comment]
@jason-gcio, @joshkimux - This ticket may no longer be relevant to the Chapter 31 form, as we no longer allow users to enter the form without first being logged in. 

<img width="1424" alt="Screen Shot 2021-09-22 at 11 22 47 AM" src="https://user-images.githubusercontent.com/15097156/134383609-da7379ed-4062-402a-b213-47f889be8b14.png">

Additionally, it looks like this may be fixed at the platform level already, [based on this code](https://github.com/department-of-veterans-affairs/vets-website/blob/699848c035d73e69b274a9ba14ba47a49f531b73/src/platform/forms/save-in-progress/SaveInProgressIntro.jsx#L187-L193), but perhaps double check with platform folks to confirm.

**Comment by ohgreatkate** [comment]
no longer relevant, the Veteran can not proceed on this form with out logging in, and then button masked as a link has been removed.
